config/base/params.env

```diff
@@ -1,9 +1,9 @@
-IMAGES_APISERVER=quay.io/opendatahub/ds-pipelines-api-server:main
-IMAGES_ARTIFACT=quay.io/opendatahub/ds-pipelines-artifact-manager:main
-IMAGES_PERSISTENTAGENT=quay.io/opendatahub/ds-pipelines-persistenceagent:main
-IMAGES_SCHEDULEDWORKFLOW=quay.io/opendatahub/ds-pipelines-scheduledworkflow:main
-IMAGES_VIEWERCRD=quay.io/opendatahub/ds-pipelines-viewercontroller:main
-IMAGES_CACHE=registry.access.redhat.com/ubi8/ubi-minimal
-IMAGES_MOVERESULTSIMAGE=registry.access.redhat.com/ubi8/ubi-micro
-IMAGES_MARIADB=registry.redhat.io/rhel8/mariadb-103:1-188
-IMAGES_DSPO=quay.io/opendatahub/data-science-pipelines-operator:main
+IMAGES_APISERVER=registry.redhat.io/rhods/odh-ml-pipelines-api-server-rhel8:latest
+IMAGES_ARTIFACT=registry.redhat.io/rhods/odh-ml-pipelines-artifact-manager-rhel8:latest
+IMAGES_PERSISTENTAGENT=registry.redhat.io/rhods/odh-ml-pipelines-persistenceagent-rhel8:latest
+IMAGES_SCHEDULEDWORKFLOW=registry.redhat.io/rhods/odh-ml-pipelines-scheduledworkflow-rhel8:latest
+IMAGES_VIEWERCRD=registry.redhat.io/rhods/odh-ml-pipelines-viewercontroller-rhel8:latest
+IMAGES_CACHE=registry.redhat.io/rhods/odh-ml-pipelines-cache-rhel8:latest
+IMAGES_MOVERESULTSIMAGE=registry.redhat.io/ubi8/ubi-micro:8.7
```

Comment:
This could also be set to `latest` right? Any reason why we locking down on this version number?

Reply:
those are images we don't build, we probably want to be more careful when be update those, test for compatibility etc. and only update on a need basis